geoprocessing/src/modules/planning-units-area-protected/dto/calculate.planning-units-area-protected.dto.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  IsOptional,
+  IsEnum,
+  IsUUID,
+  IsArray,
+  IsInt,
+  Min,
+  Max,
+} from 'class-validator';
+import { IUCNCategory } from 'src/modules/protected-areas/protected-areas.geo.entity';
+
+/**
+ * @todo We have this interface partially duplicated in the api service
+ * @file api/src/modules/projects/dto/create.project.dto.ts
+ */
+export class AreaProtectedForPlanningUnitsJob {
```

Comment:
very minor - if I understand correctly the role of this, could a name like `CalculateProtectedPortionOfPlanningUnitJob` be more accurate/explicit about the role of this job DTO?